### PR TITLE
Modified MAUI DataForm getting started content

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
Modified MAUI DataForm getting started content

### CODE REVIEW

File: MAUI/DataForm/getting-started.md [hunk 1]
```diff
@@ -7,53 +7,37 @@ control: SfDataForm
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI DataForm (SfDataForm)
+# Getting Started with the .NET MAUI DataForm
 
 This section provides a quick overview of how to get started with the [.NET MAUI DataForm(SfDataForm)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html) for .NET MAUI and a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:12]: add this line - Follow the steps below to add a basic scheduler view to your project.

File: MAUI/DataForm/getting-started.md [hunk 2]
```diff
@@ -7,53 +7,37 @@ control: SfDataForm
...
-## Creating an application using the .NET MAUI DataForm
+## Prerequisites
 
-1. Create a new .NET MAUI application in Visual Studio.
+Before proceeding, ensure the following are setup:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html), then install it.
+## Step 1: Create a New MAUI Project
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:20]: add the empty line after 20th line

File: MAUI/DataForm/getting-started.md [hunk 3]
```diff
@@ -81,6 +65,40 @@ The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.C
...
+        InitializeComponent();
+        SfDataForm dataForm = new SfDataForm();
+        this.Content = dataForm;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
 ## Creating data object
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:102]: ## Step 5: Creating data object

Comment (SriRadheshNagS) [reply MAUI/DataForm/getting-started.md:102]: Added the changes

Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:102]: after step 5, change - ## Step 6: Set data object to data form

File: MAUI/DataForm/getting-started.md (hunk 2 above)
Comment (SriRadheshNagS) [reply MAUI/DataForm/getting-started.md:20]: Added the changes

File: MAUI/DataForm/getting-started.md (hunk 1 above)
Comment (SriRadheshNagS) [reply MAUI/DataForm/getting-started.md:12]: Added the changes

File: MAUI/DataForm/getting-started.md (hunk 3 above)
Comment (SriRadheshNagS) [reply MAUI/DataForm/getting-started.md:102]: Added the changes

File: MAUI/DataForm/getting-started.md [hunk 4]
```diff
@@ -7,53 +7,40 @@ control: SfDataForm
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI DataForm (SfDataForm)
+# Getting Started with the .NET MAUI DataForm
 
 This section provides a quick overview of how to get started with the [.NET MAUI DataForm(SfDataForm)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html) for .NET MAUI and a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
 
-## Creating an application using the .NET MAUI DataForm
+Follow the steps below to add a basic dataForm view to your project.
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:14]: add in above paragraph - refer this pr
https://github.com/syncfusion-content/maui-docs/pull/2656/files

Comment (SriRadheshNagS) [reply MAUI/DataForm/getting-started.md]: Updated the changes

File: MAUI/DataForm/getting-started.md [hunk 5]
```diff
@@ -7,53 +7,38 @@ control: SfDataForm
...
-## Creating an application using the .NET MAUI DataForm
+## Prerequisites
 
-1. Create a new .NET MAUI application in Visual Studio.
+Before proceeding, ensure the following are setup:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.html), then install it.
+## Step 1: Create a New MAUI Project
```
Comment (saiganesh-sakthivel) [inline MAUI/DataForm/getting-started.md:20]: image

add the step 1 properly

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**